### ISSUE TITLE
[1.16.5] Any chance of getting framed drawers and the framing table back?

### ISSUE THREAD

**Issue by darchitectlp** [jaquadro/StorageDrawers#1055]
Really miss being able to customize my storage. Would love to see it make a comeback. Also, I'm sure you have your hands full, but I really adored your Garden Stuff mod; the aesthetics and decor were great. Would love to see that stuff make a comeback, too. Anyway, thanks for a great mod.

**Comment by SirOlrik** [comment]
I second on this, the aesthetic is a bliss to look at, knowing, I made this :)
I would also like that Controller, Slave and Compacting drawers could be framed.

**Comment by Jack-McKalling** [comment]
We've got a really nice FAQ on our discord server now. Please join, a lot of questions have been answered in there. Including framed drawers content.

**Comment by darchitectlp** [comment]
No; please stop recommending discord. Post the FAQ here or on a wiki. Discord is closed to the open net, has incredibly poor search functionality, and will cause a general disaster when it eventually closes. Info on the open net can be scanned, searched, and backed up to the way back machine; ensuring longevity. Discord is for idle chatter, not recording important information.

**Comment by jaquadro** [comment]
I can appreciate some of the concerns about discord at a broad level, I agree that in the long view it's probably a liability to the net.  On the other hand, like many other projects, I eventually adopted it to help support the project, and it's allowed other members of the community to more easily help support the project, including writing the above-mentioned FAQ.

It's okay if you don't want to utilize the discord out of principle, but there's also nothing wrong with another community member mentioning that resource.

For your benefit, I'm going to copy that answer that Jack wrote: 
`The framing table was temporarily removed from the mod, and instead, you can now use the Framed Compacting Drawers addon (it also adds the framing table and normal framed drawers) check out the pins in ⁠companion-mods`

That mod is here: https://www.curseforge.com/minecraft/mc-mods/framed-compacting-drawers

To the extent that it continues to be maintained, I'm not in a rush to pull it back directly into the mod.  Currently it's behind with no version for 1.9.3+, but I only recently got the main mod up to those versions.  There are versions for 1.16 though, as you tagged in your title.

**Comment by darchitectlp** [comment]
Thanks for your response. Totally understand that discord makes life easier in some ways. I really hope more people understand its inherent dangers, as far as being a long-term knowledge base.
